cli.js

```diff
@@ -0,0 +1,38 @@
+const main = require('./main.js');
+const _ = require('lodash');
+
+const argv = require('yargs')
+.options({
+  profile: {
+    describe: 'name of aws profile to use',
+    default: false
+  },
+  bucket: {
+    describe: 's3 bucket to upload the image to',
+    default: 's3-bucket'
```

Comment:
Committed the code for this yesterday, but was waiting on other feedback before pushing the commit (which causes Github to hide our comments pertaining to the old commit)